File: backend/DataAccess/DaoGenerated/システム共有/tt_aflogdbDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             DB操作ログテーブル
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_aflogdbDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_aflogdb";
		public long dblogseq { get; set; }                        //DB操作ログシーケンス
		public long? sessionseq { get; set; }                     //セッションシーケンス
		public string? sql { get; set; }                          //SQL
		public string? msg { get; set; }                          //メッセージ
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
    }
}

File: backend/JbdGjsService/Db/DaoGenerated/システム共有/tt_aflogdbDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             DB操作ログテーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_aflogdbDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_aflogdb"
        Public Property dblogseq As Long                        'DB操作ログシーケンス
        Public Property sessionseq As Long                     'セッションシーケンス
        Public Property sql As String                          'SQL
        Public Property msg As String                          'メッセージ
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
    End Class
End Namespace
